processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/ProcessingChainBDTopoTest.groovy

```diff
@@ -13,25 +13,19 @@ import static org.junit.jupiter.api.Assertions.*
 class ProcessingChainBDTopoTest extends ChainProcessAbstractTest{
 
     public static Logger logger = LoggerFactory.getLogger(ProcessingChainBDTopoTest.class)
+    public static String h2db = "./target/myh2gisbdtopodb;AUTO_SERVER=TRUE"
+    public static String bdtopoFoldName = "bdtopofolder"
+    public static String[] listTables = ["IRIS_GE", "BATI_INDIFFERENCIE", "BATI_INDUSTRIEL", "BATI_REMARQUABLE",
```

Comment:
If you don't need to explicitly type a variable, use `def` instead of class.

Reply:
done